release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -692,6 +692,22 @@ A platform Operator is an OLM-based Operator that can be installed during or aft
 
 For more information on platform Operators, see xref:../operators/admin/olm-managing-po.adoc#olm-managing-po[Managing platform Operators]. For more information on RukPak and its resources, see xref:../operators/understanding/olm-packaging-format.adoc#olm-rukpak-about_olm-packaging-format[Operator Framework packaging format].
 
+[id="ocp-4-12-olm-operator-node-affinity"]
+==== Controlling where an Operator is installed
+
+By default, when you install an Operator, OpenShift Container Platform installs the Operator pod to one of your worker nodes randomly.
+
+In {product-title} 4.12, you can control where an Operator pod is installed by adding affinity constraints to the Operator’s `Subscription` object.
+
+For more information, see xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-overriding-operator-pod-affinity_olm-adding-operators-to-a-cluster[Controlling where an Operator is installed].
+
+[id="ocp-4-12-olm-pod-security-label-synchronization"]
+==== Pod security admission synchronization for user-created openshift-* namespaces
+
+In {product-title} 4.12, pod security admission syncrhonization is turned on by default if an Operator is installed in user-created namespaces that have an `openshift-` prefix. Synchronization is turned on after a cluster service version (CSV) is created in the namespace. The synchronized label inherits the permissions of the service accounts in the namespace.
+
+For more information, see xref:../understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-synchronization_understanding-and-managing-pod-security-admission[Security context constraint synchronization with pod security standards].
```

Comment:
This xref is giving me a 404, but it also needs to be `.adoc` instead of `.html`

Reply:
ah! Thank you. I thought I caught that one.